### ISSUE TITLE
warmup linear problem

### ISSUE THREAD

**Issue by guanfuchen** [facebookresearch/maskrcnn-benchmark#335]
![image](https://user-images.githubusercontent.com/22321977/51070226-d95abd80-1678-11e9-8651-2bcdadbb23eb.png)

[https://github.com/facebookresearch/maskrcnn-benchmark/blob/master/maskrcnn_benchmark/solver/lr_scheduler.py#L45](url)

I use the warmup linear and constant, but get the same lr changing, the code '/' is 0 when the WARMUP_ITERS is int, is it the bug?

**Comment by fmassa** [comment]
Hi,

Are you using Python 2? This might be a Python 2-3 compatibility issue, where I'd replace
https://github.com/facebookresearch/maskrcnn-benchmark/blob/f25c6cff92d32d92abe8965d68401004e90c8bee/maskrcnn_benchmark/solver/lr_scheduler.py#L45
by
```python
alpha = float(self.last_epoch) / self.warmup_iters
```

Could you send a PR with this improvement?

**Comment by guanfuchen** [comment]
@fmassa I have create a PR, thanks for your reply.

**Comment by fmassa** [comment]
Fixed via https://github.com/facebookresearch/maskrcnn-benchmark/pull/342
